src/System.Runtime.Serialization.Xml/src/System.Runtime.Serialization.Xml.csproj

```diff
@@ -1,24 +1,14 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <AssemblyName>System.Runtime.Serialization.Xml</AssemblyName>
-    <IsPartialFacadeAssembly>true</IsPartialFacadeAssembly>
+    <IsPartialFacadeAssembly>true</IsPartialFacadeAssembly> <!-- actually it looks more like a 100% facade to me now?... -->
```

Comment:
cc @ericstj @weshaggard

Reply:
@ViktorHofer I'm going to delete that comment, but yes, I just wanted to point that out in case that means there's some way to simplify this project/assembly...

Reply:
right, that's why I pinged my colleagues who should know if there's a way to do so.

Reply:
@ericstj Thanks for clarifying!

Reply:
Maybe I can delete the `<Reference>` and `<ProjectReference>` bits of the project now though, now that there's no code which would need the references to compile.

Reply:
> Maybe I can delete the <Reference> and <ProjectReference> bits of the project now though, now that there's no code which would need the references to compile.

Well, that didn't work for some reason that I don't understand yet. I'll roll it back.

Reply:
The reason why that didn't work is that when we make it a facade, we generate some code that will create the type forwards on the assembly. If you remove all the references, then that generated code won't compile, which is why we still need at least the required references to make that generated source to compile.